src/platform/sdl/sdl_window.cpp

```
@@ -160,13 +160,14 @@ namespace moth_graphics::platform::sdl {
         SDL_SetWindowTitle(m_window, m_title.c_str());
     }
 
-    void Window::Draw() {
+    bool Window::Draw() {
         if (!m_graphics->Begin()) {
-            return;
+            return false;
```

Clang (14.0.6)

[error] redundant boolean literal in conditional return statement

(readability-simplify-boolean-expr,-warnings-as-errors)